lib/membrane_rtc_engine/endpoints/webrtc/rtp_connection_allocator.ex

```diff
@@ -108,6 +108,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator do
 
   @impl true
   def handle_cast({:bandwidth_estimation, estimation}, state) do
+    estimation = max(estimation, 300 * 1024)
```

Comment:
I have some doubts about it. It feels weirdly specific.
1. I think if you have a room with 7 peers or more with all inactive layers and no active peers, this doesn't help at all because the entire 300 kbps may be taken up the audio tracks
2. It does not address the issue of estimation being +- 1 bps in TWCC, so you're going to end up on low for ages until we probe it at the rate of at most 108% per second